### PR TITLE
Feature/task 57 reorganize widgets classes

### PR DESCRIPTION
**Pull request by MauzerTheCat**
[task-57] prepared for review

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -43,32 +43,35 @@ ext {
...
+            rxjava2           : '2.2.8',
+            rxjava2adapter    : '2.5.0',
+            rxandroid         : '2.1.1',
+            cardview          : "1.0.0",
+            retrofit          : "2.5.0",
+            logginginterceptor: "3.12.0",
+            workmanager       : "2.0.1",
+            material          : "1.0.0",
+            appcompat         : "1.0.2",
+            constraintlayout  : "1.1.3"
```
Comment (startandroid) [inline build.gradle]: Can be removed. We already have it

File: widgetsbase/build.gradle [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+}
+
+dependencies {
+    implementation fileTree(dir: 'libs', include: ['*.jar'])
+
+    implementation kotlinn.stdlibjdk8
+    implementation dagger.dagger
+    implementation androidx.appcompat
+    implementation androidx.constraintlayout
```
Comment (startandroid) [inline widgetsbase/build.gradle]: androidx_view instead of androidx

File: widgetsbase/proguard-rules.pro~HEAD [hunk 3]
```diff
@@ -0,0 +1,21 @@
+# Add project specific ProGuard rules here.
```
Comment (startandroid) [inline widgetsbase/proguard-rules.pro~HEAD]: There are 4 .gitignore and 4 proguard-rules.pro files with strange names, like: HEAD, task-57-reorganize-widgets-classes, ...
Remove them, pls.

File: build.gradle (hunk 1 above)
Comment (MauzerTheCat) [reply build.gradle]: Done

File: widgetsbase/build.gradle (hunk 2 above)
Comment (MauzerTheCat) [reply widgetsbase/build.gradle]: Done

File: widgetsbase/proguard-rules.pro~HEAD (hunk 3 above)
Comment (MauzerTheCat) [reply widgetsbase/proguard-rules.pro~HEAD]: Done

File: build.gradle (hunk 1 above)
Comment (MauzerTheCat) [reply build.gradle]: Sorry, can't get what can be removed. We use versions' names further in gradle dependencies...

Comment (MauzerTheCat) [reply build.gradle]: Done

**Pull request merged**